sockeye/training.py

```diff
@@ -587,6 +588,17 @@ def _load_trainer_states(self, fname):
         self.trainer.load_states(fname)
         logger.info('Loaded optimizer states from "%s"', fname)
 
+    def _save_lr_scheduler(self, fname):
+        with open(fname, "wb") as fp:
```

Comment:
The scheduler can also be `None`. Could you add a check for that?

Reply:
ok